tests/Unit/Models/RockTheVoteReportTest.php

```diff
@@ -8,6 +8,33 @@
 
 class RockTheVoteReportTest extends TestCase
 {
+    /**
+     * Whether the fake RTV API configuration variable is set.
+     *
+     * @var bool
+     */
+    protected $isFakerEnabled;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->isFakerEnabled = config('services.rock_the_vote.faker');
+
+        if ($this->isFakerEnabled) {
+            \Config::set('services.rock_the_vote.faker', false);
```

Comment:
Do I understand this correctly that we turn this off for testing because we want to run through the actual functionality - which uses the mocks anyhow, bypassing the actual API?

Reply:
We want to unset the "faker" flag that's used for manually testing the UI without sending actual API requests to RTV. We don't have access to a dev or QA instance of the Rock the Vote API. I wanted to add this to at least see with my own 👀 that the form worked correctly beyond just relying on the automated tests (it's burned me before).